src/functions/template-tags/post.php

```diff
@@ -30,7 +30,13 @@ function tribe_get_the_content( $more_link_text = null, $strip_teaser = false, $
 		$content = get_the_content( $more_link_text, $strip_teaser );
 	}
 
-	if ( ! doing_filter( 'the_content' ) ) {
+	$has_blocks = false;
+	// If blocks are present we need to for sure pass the content filter.
+	if ( function_exists( 'has_blocks' ) ) {
```

Comment:
Following the change suggested above, I would remove this piece of code to go with the simpler version.